frontend/model/contracts/group.js

```diff
@@ -1220,14 +1220,19 @@ sbp('chelonia/defineContract', {
     'gi.contracts/group/leaveChatRoom': {
       validate: actionRequireActiveMember(objectOf({
         chatRoomID: stringMax(MAX_HASH_LEN, 'chatRoomID'),
-        memberID: optional(stringMax(MAX_HASH_LEN), 'memberID')
+        memberID: optional(stringMax(MAX_HASH_LEN), 'memberID'),
+        // `reference` is the height used in the corresponding join action
+        reference: numberRange(1, Number.MAX_SAFE_INTEGER)
```

Comment:
The term `reference` sounds rather vague, like it could be anything.

But it seems like it must be a very specific type of value: the last known `joinedHeight` value as seen from the perspective of someone (else?) in the group. So two change requests:

1. An updated comment here to better explain what this value is, e.g. "last known `joinedHeight` value as seen from the perspective of someone else (or ourselves) in the group"
2. An updated variable name, e.g. `lastKnownJoinedHeight` or `recentJoinedHeight`